docs/components/sponsors-backers-donors.vue

```diff
@@ -0,0 +1,334 @@
+            <div class="sponsor img-thumbnail bg-light mx-auto d-flex align-items-center justify-content-center overflow-hidden">
+              <b-img-lazy :src="bronzeSponsor.imageUrl" fluid block alt="Backer image"></b-img-lazy>
+            </div>
+            <p class="sponsor-name small mb-0 pt-2">
+              <b-link
+                v-if="bronzeSponsor.website"
+                :href="bronzeSponsor.website"
+                target="_blank"
+                class="stretched-link text-break text-reset"
+              >
+                {{ bronzeSponsor.name }}
+              </b-link>
+              <template v-else>
+                {{ bronzeSponsor.name }}
+              </template>
+            </p>
+          </div>
+        </div>
+      </template>
+    </template>
+
+    <!-- BACKERS -->
+    <template v-if="backers.length > 0">
+      <h3 class="h4 mx-auto mt-4 text-muted">Backers</h3>
+      <div class="backer-container d-flex flex-wrap mx-n2 text-center justify-content-center font-weight-bold">
+        <div v-for="backer in backers" :key="backer.slug" class="m-2 position-relative">
+          <div class="backer img-thumbnail bg-light mx-auto d-flex align-items-center justify-content-center overflow-hidden">
+            <b-img-lazy :src="backer.imageUrl" fluid block alt="Backer image"></b-img-lazy>
+          </div>
+          <p class="backer-name small mb-0 pt-2">
+            <b-link
+              v-if="backer.website"
+              :href="backer.website"
+              target="_blank"
+              class="stretched-link text-break text-reset"
+            >
+              {{ backer.name }}
+            </b-link>
+            <template v-else>
+              {{ backer.name }}
+            </template>
+          </p>
+        </div>
+      </div>
+    </template>
+
+    <!-- DONORS -->
+    <template v-if="donors.length > 0">
+      <h3 class="h4 mx-auto mt-4 text-muted">Donors</h3>
+      <div class="donor-container d-flex flex-wrap mx-n1 text-center justify-content-center">
+        <div v-for="donor in donors" :key="donor.slug" class="m-1 position-relative">
+          <div class="donor img-thumbnail d-flex align-items-center justify-content-center overflow-hidden">
+            <b-link
+              v-if="donor.website"
+              :href="donor.website"
+              :title="donor.name"
+              target="_blank"
+              class="stretched-link"
+            >
+              <b-img-lazy :src="donor.imageUrl" fluid block alt="Donor image"></b-img-lazy>
+            </b-link>
+            <b-img-lazy
+              v-else
+              :src="donor.imageUrl"
+              :title="donor.name"
+              alt="Donor image"
+              fluid
+              block
+            ></b-img-lazy>
+          </div>
+        </div>
+      </div>
+    </template>
+  </div>
+</template>
+
+<style scoped>
+.sponsor {
+  width: 100px;
+  height: 100px;
```

Comment:
Should we have larger sizes based on sponsor tier level?  i.e. platinum at maybe 150px?

Reply:
Went for `140px`.